src/robot/reporting/xunitwriter.py

```diff
@@ -39,23 +39,16 @@ def __init__(self, xml_writer):
         self._writer = xml_writer
 
     def start_suite(self, suite):
-        tests, failures, skipped = self._get_stats(suite.statistics)
+        stats = suite.statistics
```

Comment:
I'm not a big fan of commenting code in general, I prefer code that can be understood without them, but here a comment like `# Accessing property only once.` would be good. Otherwise someone could refactor this code so that this assignment is inlined.